tests/components/sensorpush_cloud/conftest.py

```diff
@@ -0,0 +1,83 @@
+"""Common fixtures for the SensorPush Cloud tests."""
+
+from __future__ import annotations
+
+from collections.abc import Callable, Generator
+from typing import Any
+from unittest.mock import AsyncMock, patch
+
+import pytest
+from sensorpush_ha import SensorPushCloudApi, SensorPushCloudHelper
+
+from homeassistant.components.sensorpush_cloud.const import DOMAIN
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+
+from tests.common import MockConfigEntry
+
+
+def pytest_configure(config: pytest.Config) -> None:
+    """Register markers for tests that use data."""
+    config.addinivalue_line("markers", "data: mark test to run with specified data")
+
+
+@pytest.fixture
+def mock_api(request: pytest.FixtureRequest) -> Generator[AsyncMock]:
+    """Override SensorPushCloudApi."""
+    mock_api = AsyncMock(SensorPushCloudApi)
+    with (
+        patch(
+            "homeassistant.components.sensorpush_cloud.config_flow.SensorPushCloudApi",
+            return_value=mock_api,
+        ),
+    ):
+        yield mock_api
+
+
+@pytest.fixture
+def mock_helper(request: pytest.FixtureRequest) -> Generator[AsyncMock]:
+    """Override SensorPushCloudHelper."""
+    mock_helper = AsyncMock(SensorPushCloudHelper)
+    marker = request.node.get_closest_marker("data")
+    if marker is not None:
+        mock_helper.async_get_data.return_value = marker.args[0]
+    with (
+        patch(
+            "homeassistant.components.sensorpush_cloud.coordinator.SensorPushCloudHelper",
+            return_value=mock_helper,
+        ),
+    ):
+        yield mock_helper
```

Comment:
If you're only going to be returning the same MOCK_DATA, I would just suggest to just build that into the mock instead of building a whole fixture request framework for it